### PR TITLE
UNK hack

### PR DESCRIPTION
**Pull request by pks**

### CODE REVIEW

File: src/model_interface.cc [hunk 1]
```diff
@@ -130,6 +130,8 @@ void ModelInterface::InitializePieces() {
     }
   }
 
+  user_defined_symbols.insert("<unk>");
```
Comment (pks) [inline src/model_interface.cc:134]: This has the effect that `<unk>` tokens in the raw input fed to `spm_encode` will be encoded as UNK and not tokenized to e.g. `< unk >`.

Comment (adityanshastry) [reply src/model_interface.cc:134]: We shouldn't make this explicit as we have it defined in the configs used by the data repo here https://github.com/lilt/data/blob/a6508ecec7b57e56b1d8b08c4dd640e04a21060b/configs/bpe_operations.yaml#L18

Comment (adityanshastry) [reply src/model_interface.cc:134]: This config should be used as reference whenever training the BPE models outside of the data repo

**Review by adityanshastry — APPROVED** [review]
LGTM

**Pull request merged**